aws/sdk/examples/sqs/src/bin/hello-world.rs

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use std::process::exit;
+use tracing_subscriber::fmt::SubscriberBuilder;
+use tracing_subscriber::EnvFilter;
+
+#[tokio::main]
+async fn main() -> Result<(), sqs::Error> {
+    SubscriberBuilder::default()
+        .with_env_filter(EnvFilter::from_default_env())
```

Comment:
Turns out there's a simpler way to do this:
```rust
tracing_subscriber::fmt::init();
```
Which is equivalent: https://docs.rs/tracing-subscriber/0.2.18/src/tracing_subscriber/fmt/mod.rs.html#1127-1129